README.md

```diff
@@ -33,3 +33,15 @@ export OPERATOR_NAME=aws-efs-csi-driver-operator
 # Run the operator via CLI
 ./aws-efs-csi-driver-operator start --kubeconfig $KUBECONFIG --namespace openshift-cluster-csi-drivers
 ```
+
+
+# OLM
+
+To build an bundle + index images, use `hack/create-bundle`.
+
+```shell
+cd hack
+./create-bundle quay.io/openshift/origin-aws-efs-csi-driver:latest quay.io/openshift/origin-aws-efs-csi-driver-operator:latest quay.io/<my-repo>/efs-bundle quay.io/<my-repo>/efs-index
```

Comment:
nit: can you add a note that, these images aren't mirrored yet?

Reply:
Updated the example to CI registry with a TODO to replace it with quay once we have the images.